pkg/iac/scanners/terraform/options.go

```diff
@@ -27,14 +27,6 @@ func ScannerWithTFVarsPaths(paths ...string) options.ScannerOption {
 	}
 }
 
-func ScannerWithAlternativeIDProvider(f func(string) []string) options.ScannerOption {
-	return func(s options.ConfigurableScanner) {
-		if tf, ok := s.(ConfigurableTerraformScanner); ok {
-			tf.AddExecutorOptions(executor.OptionWithAlternativeIDProvider(f))
-		}
-	}
-}
-
```

Comment:
were `alternativeIDProviders` not in use? I'm not sure of the original motivation behind them but just want to be sure they weren't part of any use case that we might have missed.

Reply:
This is not used in the Trivy. I think it's a legacy from tfsec.

Reply:
I have noticed that there is a lot of logic in the Terraform scanner to process the scan results and a lot of it is not being used.

Reply:
> I have noticed that there is a lot of logic in the Terraform scanner to process the scan results and a lot of it is not being used.

We should remove such logic, in a separate PR.